src/components/profile/profile-qualification/certificate-form/styles.ts

```diff
@@ -28,9 +28,14 @@ export const ErrorMessage = styled(Typography)`
   font-weight: ${typography.fontWeightMedium};
 `;
 
+export const ButtonWrapper = styled('div')`
+  border-top: 1px solid var(--divider, rgba(0, 0, 0, 0.12));
```

Comment:
Could you please use a theme or consts for colors?

Reply:
fixed